tiledb/cloud/pool_manager_wrapper.py

```diff
@@ -0,0 +1,10 @@
+class PoolManagerWrapper:
```

Comment:
This class is only ever going to be useful in `client.py` and should not be consumed by external users, so it would make sense to define it as `_PoolManagerWrapper` in that file, or rename this module to start with an underscore too.

I would also add a docstring, explaining why this exists and the fact that it only implements the one method of `PoolManager` that the generated API code actually uses.

Reply:
changed